addon/helpers/ember-sparkles/axis.js

```diff
@@ -10,8 +10,13 @@ export function emberSparklesAxis([ scale ], { position, tickFormat, ticks, widt
 
   let result = axisFn().scale(scale);
 
-  if (tickFormat) {
-    result.tickFormat(timeFormat(tickFormat));
+  switch(typeof tickFormat) {
```

Comment:
Makes sense. Then in the guts of the helper we'd have 3 cases?: string, function, else?

And then d3-tick-format would be the tool to use timeFormat from d3-time-format - so axis.js no longer needs to import this.